doc/langref.html.in

```diff
@@ -9020,14 +9012,16 @@ pub const PrefetchOptions = struct {
       {#header_open|@setCold#}
       <pre>{#syntax#}@setCold(comptime is_cold: bool) void{#endsyntax#}</pre>
       <p>
-      Tells the optimizer that a function is rarely called.
+      Tells the optimizer that the current function is (or is not) rarely called.
+
+      This function is only valid within function scope.
       </p>
       {#header_close#}
 
       {#header_open|@setEvalBranchQuota#}
       <pre>{#syntax#}@setEvalBranchQuota(comptime new_quota: u32) void{#endsyntax#}</pre>
       <p>
-      Changes the maximum number of backwards branches that compile-time code
+      Increase the maximum number of backwards branches that compile-time code
       execution can use before giving up and making a compile error.
```

Comment:
Just tested it and indeed, calls to this function lowering the quota are ignored - maybe that could be explicitly noted. (`Calls that would lower the current quota are ignored.` or something.)

Reply:
The existing sentence after this one covers the details well enough I think?  Its not part of the diff, but says:
```
      If the {#syntax#}new_quota{#endsyntax#} is smaller than the default quota ({#syntax#}1000{#endsyntax#}) or
      a previously explicitly set quota, it is ignored.
```

Reply:
Ah, right, I completely overlooked that, sry.